.github/workflows/publish-containers.yml

```diff
@@ -98,11 +98,12 @@ jobs:
 
       - name: Cache Docker layers
         uses: actions/cache@v2
+        if: github.ref != 'refs/heads/main'
```

Comment:
Can you add a comment in the file describing the change? We would like to skip caching `main` as we are limited in the storage size on GH thus we prioritize 3.x branches, until `https://github.com/docker/buildx/issues/1044` is resolved we can't use image registry as cache